src/simulation/environment/eclipse/eclipse.h

```diff
@@ -51,6 +51,7 @@ class Eclipse: public SysModel {
     std::vector<ReadFunctor<SCStatesMsgPayload>> positionInMsgs;  //!< vector of msgs for each spacecraft position state for which to evaluate eclipse conditions.
     std::vector<Message<EclipseMsgPayload>*> eclipseOutMsgs;//!< vector of eclispe output msg names
     BSKLogger bskLogger;                        //!< BSK Logging
+    double REQ_CUSTOM;  //!< Custom radius, km
```

Comment:
please rename `REQ_CUSTOM` variable name to be BSK naming convention compliant.  The other names are such because this is how they were called in the planet info data base file.  How about `reqCustom`?